doc/en/user/source/webadmin/data/layers.rst

```diff
@@ -194,15 +220,23 @@ The Publishing tab is for configuring HTTP and WMS/WCS settings.
 
      .. figure:: ../images/data_layers_WFS.png
 
-        WFS otherSRS/otherCRS override
+        *WFS otherSRS/otherCRS override*
 
      The list will be used only for the capabilities document generation, but will not be used to limit the actual target SRS usage in GetFeature requests.
 
-* **WCS Settings**—Provides a list of SRSs the layer can be converted to. :guilabel:`New Request SRS` allows you to add an SRS to that list. 
+WCS Settings
+^^^^^^^^^^^^
+* **Request SRS**—Provides a list of SRSs the layer can be converted to. :guilabel:`New Request SRS` allows you to add an SRS to that list. 
 * **Interpolation Methods**—Sets the raster rendering process, if applicable.
 * **Formats**—Lists which output formats a layers supports.
 * **GeoSearch**—When enabled, allows the Google Geosearch crawler to index from this particular layer. See `What is a Geo Sitemap? <http://www.google.com/support/webmasters/bin/answer.py?hl=en&answer=94554>`_ for more information.
```

Comment:
This is left unchanged from the current doc.  I didn't check too hard for correctness - this patch was mostly focussed on formatting and clarification.  Suggest leaving as is for now, it can be removed later if needed.